apps/sasquatch/src/main/java/com/microsoft/appcenter/crashes/CrashesPrivateHelper.java

```diff
@@ -5,14 +5,17 @@
 
 package com.microsoft.appcenter.crashes;
 
+import com.microsoft.appcenter.crashes.ingestion.models.ErrorAttachmentLog;
+
 import java.util.Map;
 
 public final class CrashesPrivateHelper {
 
     private CrashesPrivateHelper() {
     }
 
-    public static void trackException(Throwable throwable, Map<String, String> properties) {
+    public static void trackException(Throwable throwable, Map<String, String> properties, Iterable<ErrorAttachmentLog> attachments) {
```

Comment:
This no longer exists on the track exception branch. This feature must be based on `feature/track-exception` branch.